eli5/sklearn/text.py

```diff
@@ -82,8 +96,12 @@ def _get_weighted_spans_from_union(doc, vec_union, feature_weights):
     named_found_features = []
     for vec_name, vec in vec_union.transformer_list:
         vec_prefix = '{}__'.format(vec_name)
-        feature_fn = lambda x: (
-            x[len(vec_prefix):] if x.startswith(vec_prefix) else None)
+
+        def feature_fn(x):
+            if (not isinstance(x, FormattedFeatureName)
```

Comment:
Right, there were no test for feature union text highlighting where missing features appeared, I added missing test in https://github.com/TeamHG-Memex/eli5/pull/124